src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs

```diff
@@ -708,5 +709,195 @@ public static nuint RotateRight(nuint value, int offset)
             return (nuint)RotateRight((uint)value, offset);
 #endif
         }
+
+        private static class Crc32Fallback
```

Comment:
Nit: I'm not sure about the BCL style preferences around nested classes, but I'd personally put this at the end of the parent rather than having it hanging out in the middle. Area owners might prefer this to be split into another file, though 🤷‍♂️

Reply:
@tannergooding What do you think 😄?